staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -493,12 +504,16 @@ func (c completedConfig) New(name string, delegationTarget DelegationTarget) (*G
 		preShutdownHooks:       map[string]preShutdownHookEntry{},
 		disabledPostStartHooks: c.DisabledPostStartHooks,
 
-		healthzChecks: c.HealthzChecks,
+		healthzChecks:              c.HealthzChecks,
+		readyzChecks:               c.ReadyzChecks,
+		readinessStopCh:            make(chan struct{}),
```

Comment:
does the wiring really need these 3 in the config or would GenericAPIServer be enough?

Reply:
I'm not sure I follow what you are saying.. the portion of the code you are commenting on is in fact the GenericAPIServer. Specifically this is where we initialize the GenericAPIServer object.